### PR TITLE
Translate 2020 Fukuoka Ruby Award Competition (id)

### PR DESCRIPTION
**Pull request by meisyal**
cc: @kuntoaji

### CODE REVIEW

File: id/news/_posts/2019-10-16-fukuoka-ruby-award-2020.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Anda untuk mengikuti kompetisi Ruby berikut. Jika Anda pernah mengembangkan
+sebuah program Ruby yang menarik, dianjurkan untuk mengikuti kompetisi ini.
+
+2020 Fukuoka Ruby Award Competition - Hadiah Utama - 1 Juta Yen!
+
+Batas akhir pendaftaran: 11 Desember 2019
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz dan sebuah grup panelis akan memilih pemenang kompetisi ini. Haidah utama
```
Comment (kuntoaji) [inline id/news/_posts/2019-10-16-fukuoka-ruby-award-2020.md]: typo, "Hadiah" not "Haidah"

File: id/news/_posts/2019-10-16-fukuoka-ruby-award-2020.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+dari kompetisi ini adalah 1 juta Yen. Hadian pemenang sebelumnya termasuk
```
Comment (kuntoaji) [inline id/news/_posts/2019-10-16-fukuoka-ruby-award-2020.md]: typo again, "Hadiah" not "Hadian"

File: id/news/_posts/2019-10-16-fukuoka-ruby-award-2020.md (hunk 1 above)
Comment (meisyal) [reply id/news/_posts/2019-10-16-fukuoka-ruby-award-2020.md]: Thank you for catching that. Fixed @kuntoaji

File: id/news/_posts/2019-10-16-fukuoka-ruby-award-2020.md (hunk 2 above)
Comment (meisyal) [reply id/news/_posts/2019-10-16-fukuoka-ruby-award-2020.md]: Fixed @kuntoaji

**Pull request merged**